Assets/Scripts/Models/Buildable/Furniture.cs

```diff
@@ -730,6 +748,29 @@ public void ReadXmlBuildingJob(XmlReader reader)
     }
 
     /// <summary>
+    /// Reads the XML building job.
+    /// </summary>
+    /// <param name="reader">The XML reader to read from.</param>
+    public void ReadXmlDeconstructJob(XmlReader reader)
```

Comment:
Instead of storing the job params you just create the deconstruct job with a null Tile, and add it to the PrototypeManager (check how is done for the build job). Then when you are going to deconstruct, you grab the job prototype, clone it and set the tile.